src/server/Dockerfile

```diff
@@ -31,11 +38,12 @@ RUN chmod -R 777 /app/static
 COPY . /paws-data-pipeline/server
 
 RUN chmod 777 /paws-data-pipeline/server
-
+# RUN ufw allow 5000
 WORKDIR /app
 
+# --uid uwsgi
+CMD uwsgi --http-socket :5000 --plugin python38 --module wsgi:app --chdir /app --pythonpath .
```

Comment:
The `uwsgi` command string needs go at the end of the current CMD line, after the alembic command (replacing 'python -m ...'